### DISCUSSION TITLE
chatgpt 官网挂了？

### DISCUSSION THREAD

**Discussion by xiaoguaishou92** [linweiyuan/go-chatgpt-api#75] [General]
chatgpt 官网挂了？现在返回500

**Comment by linweiyuan** [comment]
没有遇到，官网和 API 都正常 [OpenAI Status](https://status.openai.com)

**Reply by xiaoguaishou92** [reply]
![image](https://user-images.githubusercontent.com/20310407/233908471-b58ca3c1-0050-4de6-a00b-6fcfc873a611.png)
z早上挂了一阵，我看现在好了

**Reply by linweiyuan** [reply]
图片这个错是 403（CF 验证过期了），打开页面两分钟不动再发请求必出现这个错然后网页自动刷新

**Reply by xiaoguaishou92** [reply]
不是，这个是在官网的截图。。。不是从镜像站点的

**Reply by linweiyuan** [reply]
就是指官网

**Reply by linweiyuan** [reply]
貌似有时也不会自动刷新

https://user-images.githubusercontent.com/49076004/233914173-f82b0ada-2df0-47ab-a581-e817fe72cf3c.mp4

自动刷新的场景

https://user-images.githubusercontent.com/49076004/233533755-29d5dc49-cf8f-46b1-b369-9c25a63a985d.mp4

**Comment by xiaoguaishou92** [comment]
![image](https://user-images.githubusercontent.com/20310407/233909430-d3a214b3-57a7-43df-b58a-4054cccf7a21.png)
